src/csharp/Microsoft.Spark.E2ETest/IpcTests/Sql/DataFrameTests.cs

```diff
@@ -458,6 +459,7 @@ public void TestSignaturesV2_3_X()
             _df.Explain(false);
 
             Assert.Equal(2, _df.Columns().ToArray().Length);
+            Assert.IsAssignableFrom<IEnumerable<Tuple<string, string>>>(_df.DTypes());

```

Comment:
Since this is not an IPC call, can we check the values returned by `DTypes`?

Reply:
I am thinking if there is another way to test the values besides just calling `Schema().Fields` on the dataframe and iterating over that?

Reply:
We already know what the schema is for this test.  Construct a List of tuples and compare it to that.  

```csharp
    var expected = new Tuple<string, string>[] {
      new Tuple<string, string>("name", "string"),
      new Tuple<string, string>("age", "int")
    };
```